src/components/dev-hub/card-list.js

```diff
@@ -38,12 +40,16 @@ export default ({ items = [], limit = 9 }) => {
             <CardContainer>
                 {items.slice(0, visibleCards).map(item => (
                     <ArticleCard
-                        key={item['meta-description']}
-                        image={item['atf-image']}
-                        tags={item.languages}
-                    >
-                        {item['meta-description']}
-                    </ArticleCard>
+                        key={item['_id']}
+                        image={withPrefix(item['atf-image'])}
```

Comment:
I can't think of a great way to check images locally, was running into a similar problem with articles.